docker/images/dev/Dockerfile

```diff
@@ -18,3 +18,6 @@ RUN apt-get update && \
 
 # Install development dependencies
 RUN pip install .[dev]
+
+# Forcibly remove the marsha package
+RUN pip uninstall marsha
```

Comment:
Group it with the previous run to limit the number of layer

```
RUN pip install .[dev] && \
    pip uninstall marsha
```

Reply:
Oups, thank you